### PR TITLE
List available features

### PR DESCRIPTION
**Pull request by JTurbakiewicz**

### CODE REVIEW

**Comment by JTurbakiewicz** [comment]
Very simple script listing the rules present in the repository. Output is a CSV file, which can be parsed to Excel quite nicely:

image

I can make an .exe file out of the script of course.

**Review by Ghesselink — CHANGES_REQUESTED** [review]
Looks good. 

Should we also add a recent version of the .csv output in the repository?

**Review by Ghesselink — APPROVED** [review]

**Comment by Ghesselink** [comment]
@evandroAlfieri That makes sense. It happens quite often already that a PR evolves to a bigger discussion and then remains open indeed .. Let's merge it then 

@JTurbakiewicz I've approved the changes. If I'm right you can merge the PR when you resolve the conversations (1 approving review).

File: features/resources/list_repository/_list_rules.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                information_about_feature["rule_description"] = "".join(
+                    feature_desc.split("\n")[1:]
+                )
+            else:
+                information_about_feature["rule_description"] = ""
+
+            lines = file_content.split("\n")
+            at_lines = [line.lower() for line in lines if line.startswith("@")]
+
+            tags = ["@disabled", "@implementer-agreement", "@informal-proposition"]
```
[the remark this answers is not in the document]
Comment (JTurbakiewicz) [reply features/resources/list_repository/_list_rules.py:31]: I agree with the comment. At the moment I will resolve it based on Evandros good point, but in the future something to think about.

File: features/resources/list_repository/_list_rules.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            for tag in tags:
+                information_about_feature[tag[1:]] = tag in at_lines
+
+            rule_tag = [item for item in at_lines if item not in tags]
+            information_about_feature["rule_tag"] = rule_tag[0].upper()[1:]
```
[the remark this answers is not in the document]
Comment (JTurbakiewicz) [reply features/resources/list_repository/_list_rules.py:36]: I added a simple check len(item) == 4, eg. @als

Obviously that's just a super simple check, but keeping with the spirit of merging quick, should work just fine.

**Pull request merged**